apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverview.tsx

```diff
@@ -209,6 +210,19 @@ export const SignedVerdictOverview: React.FC = () => {
     }
   }
 
+  const canCaseFilesBeOpened = () => {
+    if (
+      workingCase?.accusedAppealDecision === CaseAppealDecision.APPEAL ||
+      workingCase?.prosecutorAppealDecision === CaseAppealDecision.APPEAL ||
+      !!workingCase?.accusedPostponedAppealDate ||
+      !!workingCase?.prosecutorPostponedAppealDate
```

Comment:
This is not aligned with the server. An open button may be shown even if the server rejects the open request.